src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private static int salary;
```

Comment:
If you are counting something in your method for example using some variable for the result, don’t make this variable a class field.
The class fields should represent the state of the class objects otherwise they should be local variables.
давай перемітимо змінну salary з полів класу в метод, де вона безпосередньо використовується